tests/garage/tf/core/test_gru.py

```diff
@@ -0,0 +1,384 @@
+from nose2.tools.params import params
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.core.gru import gru
+from tests.fixtures import TfGraphTestCase
+from tests.helpers import recurrent_step_gru
+
+
+class TestGRU(TfGraphTestCase):
+    def setUp(self):
+        super().setUp()
+        self.batch_size = 2
+        self.hidden_dim = 2
+
+        self._step_hidden_var = tf.placeholder(
+            shape=(self.batch_size, self.hidden_dim),
+            name='initial_hidden',
+            dtype=tf.float32)
+
+        self.gru_cell = tf.keras.layers.GRUCell(
+            units=self.hidden_dim,
+            activation=tf.nn.tanh,
+            kernel_initializer=tf.constant_initializer(1),
+            recurrent_activation=tf.nn.sigmoid,
+            recurrent_initializer=tf.constant_initializer(1),
+            name='lstm_layer')
+
+    @params(
+        (1, 1, 1, 0, 0),        # yapf: disable
+        (1, 1, 3, 0, 0),        # yapf: disable
+        (1, 3, 1, 0, 0),        # yapf: disable
+        (3, 1, 1, 0, 0),        # yapf: disable
+        (3, 3, 1, 0, 0),        # yapf: disable
+        (3, 3, 3, 0, 0),        # yapf: disable
+        (1, 1, 1, 0.5, 0.5),    # yapf: disable
+        (1, 1, 3, 0.5, 0.5),    # yapf: disable
+        (1, 3, 1, 0.5, 0.5),    # yapf: disable
+        (3, 1, 1, 0.5, 0.5),    # yapf: disable
+        (3, 3, 1, 0.5, 0.5),    # yapf: disable
+        (3, 3, 3, 0.5, 0.5))    # yapf: disable
```

Comment:
i think you should be able to just put one yapf_disable right after the `@params` statement and it should ignore the whole block.

if it doesn't, you can use this form
```python
# yapf: disable
stuff()
more_stuff()
# yapf: enable
```